src/coordinator/functions/clamp.go

```diff
@@ -93,19 +93,27 @@ type ClampNode struct {
 
 // Process the block
 func (c *ClampNode) Process(ID parser.NodeID, b block.Block) error {
```

Comment:
Should clamp node be transformed into a lazy function?

Reply:
yea planning in another diff.

Reply:
added a TODO